.github/workflows/build.yml

```diff
@@ -126,15 +126,16 @@ jobs:
         uses: MansaGroup/nrwl-nx-action@v2
         with:
           targets: "build"
-          projects: "winglang,sdk,wing-api-checker,jsii-docgen,wing-playground"
```

Comment:
It seems like this list of projects effectively builds all the projects. no?

Reply:
It's specifically leaving out the native-based projects (vscode and lsp)

Reply:
So once this is merged, it'll take a lot longer

Reply:
I'll exclude them.

Reply:
For the purpose of this PR, I don't think this line needs to be changed

Reply:
added "exclude"